api/envoy/extensions/filters/http/file_system_buffer/v3/file_system_buffer.proto

```diff
@@ -84,23 +85,24 @@ message StreamConfig {
   // ``memory_buffer_bytes_limit`` was 32MiB, and ``storage_buffer_queue_high_watermark_bytes``
   // was 64MiB, and the filesystem is backed up so writes are not occurring promptly,
   // then:
+  //
   // * Any request less than 32MiB will eventually pass through without ever attempting
   //   to write to disk.
   // * Any request with over 32MiB buffered will start trying to write to disk.
-  //   * If it reaches (32+64)MiB buffered and not yet written to disk, a high
-  //     watermark signal is sent to the source.
+  //   If it reaches (32+64)MiB buffered in memory (write to disk isn't keeping up), a high
+  //   watermark signal is sent to the source.
   // * Any stream whose total size exceeds
   //   ``memory_buffer_bytes_limit + storage_buffer_bytes_limit`` will provoke an error.
   //   (Note, if the recipient *is* consuming data then it is possible for such an
   //   oversized request to pass through the buffer filter, provided the recipient
   //   isn't consuming data too slowly.)
   //
   // The low watermark signal is sent when the memory buffer is at size
-  //  ``memory_buffer_bytes_limit + (storage_buffer_queue_high_watermark_bytes / 2)``.
+  // ``memory_buffer_bytes_limit + (storage_buffer_queue_high_watermark_bytes / 2)``.
   google.protobuf.UInt64Value storage_buffer_queue_high_watermark_bytes = 4;
 }
 
-// [#not-implemented-hide:]
+// FileSystemBufferFilterConfig :ref:`configuration overview <config_http_filters_file_system_buffer>`.
 // Configuration for file system buffer filter.
```

Comment:
nit: I think this can be merged to a single sentence that would make it easier to read:
```
A :ref:`file system buffer <config_http_filters_file_system_buffer>` filter configuration.
```

Reply:
Done.